packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -221,9 +221,10 @@ const argv = yargs
       describe: 'More verbose logging',
       type: 'boolean',
     },
-    'noWatchman': {
-      describe: 'Do not use watchman when not in watch mode',
+    'watchman': {
+      describe: 'Use watchman when not in watch mode',
```

Comment:
I agree that it's confusing that you could specify that watchman shouldn't be used but also that you want to use watch mode. I think in that scenario it's better to stop and report an error instead of disregarding wishes to not use watchman